test/pkg/debug/collector.go

```diff
@@ -142,13 +142,17 @@ func PrintTestTimes(times map[string][]TimeInterval) map[string][][]string {
 }
 
 // WriteTimestamps will create a temp directory and a .csv file for each suite test
-func WriteTimestamps(path string, timestamps *TimeIntervalCollector) error {
-	directory, err := os.MkdirTemp("/tmp", "")
-	if err != nil {
-		return err
+// If the OUTPUT_DIR environment variable is set, we'll print the csvs to that directory.
+func WriteTimestamps(outputDir string, timestamps *TimeIntervalCollector) error {
+	var err error
+	if outputDir == "" {
+		outputDir, err = os.MkdirTemp("/tmp", "")
+		if err != nil {
+			return err
+		}
 	}
 	for name, table := range PrintTestTimes(timestamps.suiteTimeIntervals) {
-		file, err := os.CreateTemp(directory, fmt.Sprintf("*-%s.csv", name))
+		file, err := os.CreateTemp(outputDir, fmt.Sprintf("*-%s.csv", name))
```

Comment:
Should you be using createtmp if someone specifies a directory that they want to be keeping around? I'm not sure what the impact of adding a temp file is like this if someone expects the file to live for a longer period of time. I believe that the kernel can come by later and cleanup the file if it's marked as temporary

Reply:
I think the best we could do here is inferring if the string starts with `/tmp` that it's a temporary directory, and otherwise isn't. I'm expecting this to be containerized and ephemeral, and I'd expect automation to copy these files elsewhere for permanent storage. I also think files in /tmp/ are cleaned up after 10d anyway.

If the /tmp check is good enough for you, I don't mind adding that in

Reply:
Going to resolve this and remove the hold since I think the expectation is that we'll be only using these files temporarily for now. We can always add the functionality in later.